test/integration/targets/module_utils/runme.sh

```diff
@@ -4,5 +4,8 @@ set -eux
 
 ANSIBLE_ROLES_PATH=../ ansible-playbook module_utils_basic_setcwd.yml -i ../../inventory "$@"
 
-ansible-playbook module_utils_test.yml -i ../../inventory -v "$@"
+# Keep the -vvvvv here. This acts as a test for testing that higher verbosity
+# doesn't traceback with unicode in the custom module_utils directory path.
+ansible-playbook module_utils_test.yml -i ../../inventory -vvvvv "$@"
```

Comment:
Since this playbook has multiple tasks, can we create a smaller playbook with a single task to increase the verbosity on? That way we'll minimize the noise when running the tests.